src/Runtime/ExecutionSession.cpp

```diff
@@ -26,6 +26,8 @@
 
 #include "ExecutionSession.hpp"
 
+void omTensorListDestroyWithoutDestroyingTensors(OMTensorList *list);
```

Comment:
Maybe move into `ExecutionSession.hpp`?

Reply:
I don't want to expose it to anyone doing the ExecutionSession, but created a OMTensorListHelper.hpp file for that.

Reply:
moved it in a header file about om tensor list helpers